.github/workflows/deploy-vagrant.yaml

```
@@ -0,0 +1,39 @@
+name: Single node x1 with vagrant, Ubuntu, kind
```

It's not single node
```suggestion
name: Multi node cluster with Vagrant, Ubuntu, Kubespray
```

---

fixed